src/System.Private.ServiceModel/src/System/ServiceModel/Channels/RTHttpHandlerToFilter.cs

```diff
@@ -0,0 +1,262 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Same as above, this isn't a partial implementation with netcore only code in this file. This is the entire implementation.